dp3t-sdk/sdk/src/main/java/org/dpppt/android/sdk/DP3T.java

```diff
@@ -271,7 +270,7 @@ private static void executeIAmInfected(Activity activity) {
 
 							try {
 								appConfigManager.getBackendReportRepository(activity)
-										.addGaenExposee(exposeeListRequest, pendingIAmInfectedRequest.exposeeAuthMethod,
+										.addGaenExposeeBlocking(exposeeListRequest, pendingIAmInfectedRequest.exposeeAuthMethod,
```

Comment:
this should not be blocking, because executeIAmInfected() might be run from the UI thread.